syft/pkg/cataloger/dotnet/cataloger.go

```diff
@@ -0,0 +1,14 @@
+package dotnet
+
+import (
+	"github.com/anchore/syft/syft/pkg/cataloger/common"
+)
+
+// NewDotnetDepsCataloger returns a new Dotnet cataloger object base on deps json files.
+func NewDotnetDepsCataloger() *common.GenericCataloger {
+	globParsers := map[string]common.ParserFn{
+		"**/*deps.json": parseDotnetDeps,
```

Comment:
Should there be an extra dot here?
```suggestion
		"**/*.deps.json": parseDotnetDeps,
```

Reply:
Yes, that's missing